### ISSUE TITLE
Update documentation and code to stay up to date with new phabricator version 

### ISSUE THREAD

**Issue by paladox** [bloomberg/phabricator-tools#81]
Hi please update phabricator and documentation since phabricator upstream has been upgraded and your using a really outdated version.

**Comment by aevri** [comment]
Hi!

Yup agreed, unfortunately the Docker and Puppet builds are tracking revisions from a couple of years back; it's still pretty great back then too though :)

Are you using phabricator-tools currently or are you saying you'd be interested to use if it was tracking a more recent version?

Cheers!
Angelos

**Comment by paladox** [comment]
@aevri yes, I'm trying to use it but using newer phabricator version is not working. The Wikimedia foundation would like to use it. https://phabricator.wikimedia.org/T132863 but first we have to test it. and phabricator.wikimedia.org is using the new phabricator version.

and thanks for replying.

**Comment by paladox** [comment]
certificates were removed and replaced with conduit tokens I think.

**Comment by paladox** [comment]
I installed using puppet

**Comment by aevri** [comment]
Aha, this is interesting! :) I'm on holiday at the moment, will have a think about this when I'm back in office. I see @20after4 has started looking into it, hopefully you've got enough that you can start evaluating phabricator-tools against your version of Phabricator.

I imagine you might also wish for more documentation around starting with arcyd, it's been used heavily in production by us for some time but I think we maintain the only instance so there hasn't been as much demand for getting started guides.

The docs in this dir are probably the best at the moment:
https://github.com/bloomberg/phabricator-tools/blob/master/doc/man/arcyd/arcyd.generated.txt

This test script may give you more detail on how to get started:
https://github.com/bloomberg/phabricator-tools/blob/master/testbed/arcyd/exercise_arcyd.sh

**Comment by paladox** [comment]
Ok.

**Comment by paladox** [comment]
@aevri Hi how does this work. How do I install the tools and where do I install them. how do I track the repo's. I have the repos in /var/repo/**. But it starts but pushing the changes still pushes them inside a branch instead of making a diff.

**Comment by paladox** [comment]
Does this work with ssh only or can it work with http/https please.

**Comment by aevri** [comment]
Heya!

> Does this work with ssh only or can it work with http/https please.

I think this question is about Arcyd's use of Git. Arcyd simply passes urls on to Git, so it supports http(s), ssh, file system, the Git protocol, etc. The Git configuration of the user running the Arcyd daemon will determine the behaviour of Git.

**Comment by paladox** [comment]
Ok thanks. I've followed your guide. It dosent tell you have to install the tools. But I'm stuck at how to get it working. I'm not sure if it is even working for me.

**Comment by aevri** [comment]
Yeah there's not much in the way of a getting started guide for Arcyd yet.
Will see what I can do when back from hols, hopefully you'll still be
interested then :) As I'm sure you know, the great thing about it is that
you only have to configure the 'server side' once and then all your clients
only need to worry about using Git.

**Comment by paladox** [comment]
@aevri Hi yep and thanks. I am testing it and seeing if it works for Wikimedia foundation which per the task I linked above would like to use this since we use gerrit.wikimedia.org currently which uses git but moving over to phabricator will cause them to have to also use arc with php. This project will resolve that :). 

Do we have to setup staging repo or can we start it right away without needing to setup blueprint.
